UnitystationLauncher/Models/Api/Server.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.IO;
+using System.Net;
```

Comment:
This Codacy issue is a false positive, removing this causes the compile to fail. It is being used on line 63 below.